book/command-line-parsing.md

```diff
@@ -44,7 +44,17 @@ a file, applies the MD5 one-way cryptographic hash function to the data, and
 outputs an ASCII hex representation of the result: [MD5 one-way cryptographic
 hash function]{.idx}[command-line parsing/basic approach to]{.idx}
 
-<link rel="import" href="code/command-line-parsing/md5/md5.ml" />
+```ocaml
```

Comment:
I'm very confused by this example: The source here is an ml, not an mlt, but it's not obvious to me how I would know that. I would assume that I would need some metadata here to keep track of the location of the file that this example comes from. 

What happens in the workflow if I edit the text here? Does it get reverted next time I build, or does it flow the other way, causing the shell examples later to change to reflect running this new code?